about.html

```diff
@@ -7,57 +7,93 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
+    <header>
+        <span><img  class="aboutpic"src="https://github.com/LambdaSchool/web-Sprint-Challenge-User-Interface-and-Git/blob/master/img/lambda-black.png?raw=true"></span>
+        
```

Comment:
Good use of the span tag, another way to make the code more semantic and apply good UI/UX design which most website home pages allow you to click the IMG and return to the home screen. 
EX:
<nav>
<a href="idex.html"><img src="img URL"></a>